dockerfiles/base/scripts/base/commands/cmd_config.sh

```diff
@@ -29,11 +29,14 @@ images will use those binaries instead of their embedded ones.\n"
 }
 
 pre_cmd_config() {
-  true
+  :
 }
 
-cmd_config() {
+post_cmd_config() {
+  :
```

Comment:
It works both ways - you don't have to have empty methods, but I found it cleaner to have empty methods for all of the base just to be pedantic.

You don't need these in any CLI assemblies built from base.